front/Makefile

```diff
@@ -0,0 +1,202 @@
+	@echo "$(NC)\npress any key to continue"
+	@bash -c "read -n 1 -s -r";
+	@tput cuu1 && tput el;
+	@echo "[STEP 4/4] $(GREEN)'make stop$(NC)' to stop the daemon (container)"
+	@echo "$(NC)\npress any key to continue"
+	@bash -c "read -n 1 -s -r";
+	@tput cuu1 && tput el; 
+	@echo "$(RED)[OTHER STUFF] Before commiting:$(NC)"
+	@echo "\t'$(GREEN)make linter$(NC)' to check norm (app must be installed)"
+	@echo "\tIf linter show an error:\n\t\tYou think\
+	 it is a anti-pattern, feel free to submit a issue about the linter\n\t\t\
+	 add this line before the error // eslint-disable-next-line\n\t\t\
+	 or disable eslint in the entire file with  /* eslint-disable */ \n\t\t\
+	 this will be checked by the CI team Mates ('make check_norm_bypass' target FIX will be available soon)"
+	@echo "$(NC)\npress any key to continue"
+	@bash -c "read -n 1 -s -r";
+	@tput cuu1 && tput el; 
+	@echo "[LAST STEP BEFORE CODE]\nIf new node package has been added to the app: (or if you want just to clean)"
+	@echo "\t'make clean' to remove node_modules and package-lock.json and re-install the app with 'make install'"
+	@echo "$(NC)\npress any key to continue"
+	@bash -c "read -n 1 -s -r";
+	@tput cuu1 && tput el;
+
+already_launch_serve:
+ifneq ($(shell docker ps --filter "name=frontend" -q),)
+	@echo "killing previous service..."
+	@make -s -C . stop
+endif
+
+stop:
+	@docker kill frontend
+
+# WIP DEPENDANCY - NEED SEQUENCE FIX 
+warn_check_norm_bypass_agreement:
+	@echo "You are about to check the norm bypass, it will uninstall the app you need to re-install it after"
+	@echo "Do you want to continue ? [y/n]"
+	@read -r answer; \
+	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
+		make -s -C . do_norm_bypass_check 2>/dev/null; \
+		echo "Norm bypass checked"; \
+	else \
+		echo "Aborting"; \
+		err "aborting";\
+	fi
+	@echo -n "Number of reported eslint rules : "
+	@cat norm.report | wc -l
+	@echo "Do you want to see report ? [y/n]"
+	@read -r answer; \
+	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
+		cat ./norm.report; \
+	fi
+	@echo "Keep report file norme.report ? [y/n]"
+	@read -r answer; \
+	if [ "$$answer" = "y" ] || [ "$$answer" = "Y" ]; then \
+		echo "File keeped"; \
+	else \
+		make -s -C . clean; \
+	fi
+
+do_norm_bypass_check:
+	@make -s -C . clean
+	@grep -rw  "// eslint-disable-next-line" ./app > ./norm.report
+	@grep -rw  "eslint-disable" ./app >> ./norm.report
+
+check_norm_bypass: warn_check_norm_bypass_agreement
+
+linter:
+ifeq ($(DEV_APP_INSTALLATION_LOCKER), 0)
+	@make -s -C . start_container
+	@echo "cd /app && npm run lint" | docker exec -i frontend /bin/bash
+	docker kill frontend
+else
+	@echo "Please install the dev server with 'make install' to have the benefit of the linter"
+endif
+
+attach_run_serve:
+	@echo "cd /app && npm run serve" | docker exec -i frontend /bin/bash
+
+create_container:
+ifeq ($(shell docker ps --all --filter "name=frontend" -q),)
+	@echo "Container doesnt exist"
```

Comment:
"s" missing in exists :)

Reply:
I can't see the error, can you tell me the right spelling ?